bin/travis-build.bash

```diff
@@ -15,29 +14,19 @@ echo "CKAN branch: $latest_ckan_release_branch"
 git checkout $latest_ckan_release_branch
 python setup.py develop
 pip install -r requirements.txt --allow-all-external
-pip install -r dev-requirements.txt --allow-all-external
 cd -
 
 echo "Creating the PostgreSQL user and database..."
 sudo -u postgres psql -c "CREATE USER ckan_default WITH PASSWORD 'pass';"
 sudo -u postgres psql -c 'CREATE DATABASE ckan_test WITH OWNER ckan_default;'
 
-echo "SOLR config..."
-# Solr is multicore for tests on ckan master, but it's easier to run tests on
-# Travis single-core. See https://github.com/ckan/ckan/issues/2972
-sed -i -e 's/solr_url.*/solr_url = http:\/\/127.0.0.1:8983\/solr/' ckan/test-core.ini
-
 echo "Initialising the database..."
 cd ckan
 paster db init -c test-core.ini
 cd -
 
 echo "Installing ckanext-notify and its requirements..."
 python setup.py develop
-pip install -r dev-requirements.txt
-
-echo "Moving test.ini into a subdir..."
-mkdir subdir
-mv test.ini subdir
```

Comment:
Are there any configurations specific to ckanext-notify that we'd want to make sure are in the config file? e.g making sure the plugin is activated in the `plugins` variable?

Example: https://github.com/ckan/ckanext-harvest/blob/master/test.ini#L18

Reply:
Making sure the plugin is activated is optimal. I will create a test.ini file. The tests do not really require any specific configurations. I'll create a test.ini file similar to this: 
https://github.com/andela/ckanext-datarequests/blob/master/test.ini